frontend/src/static/css/includes/typography/_typography.scss

```
@@ -2,6 +2,7 @@
 // @import './_gfonts.scss';
 
 @import './_mixins_typoghraphy.scss';
+@import './_text_styles.scss';
```

Stylelint (17.7.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)